qa/tasks/vstart_runner.py

```diff
@@ -522,7 +529,7 @@ def safe_kill(pid):
     os.kill annoyingly raises exception if process already dead.  Ignore it.
     """
     try:
-        return os.kill(pid, signal.SIGKILL)
+        return os.system('sudo kill -9 {}'.format(pid))
```

Comment:
We want to avoid using `sudo` anywhere in `vstart.sh` or in `vstart_runner.py`. Why is this necessary?

Reply:
I'll get "permission denied" error message otherwise.

Reply:
@rishabh-d-dave I think I guided you in the wrong direction. Using `sudo` in front of vstart_runner.py is not going to be viable in my own testing because it messes up the permissions for everything vstart_runner creates/touches.

With that said, I think it's possible to use `--setuid` for `nsenter` so that the uid is not root. Then you can kill it normally.